packages/shared/src/schema/user/put.ts

```diff
@@ -1,6 +1,15 @@
 import { z } from 'zod';
 
-const nameSchema = z.string().min(2).max(50);
+export const PutValidation = {
```

Comment:
Is this a better way to do this or just preference?

Reply:
The UI will need the values to help guide the user _**before**_ a validation failure occurs... do you have a better idea for naming? I struggled with this a bit and just settled on this pattern but happy to change it if you have a better recc.